CMakeLists.txt

```diff
@@ -270,27 +270,13 @@ add_dependencies(move_artifacts copymodulemap)
 
 # Provide our own modulemap for implementations other than libcxx.
 if (NOT WIN32 AND NOT libcxx)
-  # Write a empty overlay file to the output directory that CMake can run its compiler tests.
-  # We will create the actual overlay later in the configuration.
-  file(WRITE ${CMAKE_BINARY_DIR}/include/modulemap.overlay.yaml "{'version' : 0, 'roots' : []}")
-  set(__vfs_overlay "-ivfsoverlay ${CMAKE_BINARY_DIR}/include/modulemap.overlay.yaml")
-
-  if (cxxmodules)
-    set(ROOT_CXXMODULES_COMMONFLAGS "${ROOT_CXXMODULES_COMMONFLAGS} ${__vfs_overlay}")
-  endif()
 
   # Only use the first path from the HEADERS_LOCATION (which is separated by colons).
   get_property(__libcpp_full_paths GLOBAL PROPERTY ROOT_CLING_CXX_HEADERS_LOCATION)
   string(REGEX MATCHALL "[^:]+" __libcpp_full_paths_list "${__libcpp_full_paths}")
   if (__libcpp_full_paths_list)
     list(GET __libcpp_full_paths_list 0 __libcpp_full_path)
```

Comment:
You can also delete this whole block up of code as it's no longer needed (from here until `# Only use the first ...`).